src/Utils.js

```diff
@@ -17,7 +17,13 @@ function debugAddingSkill(skillName) {
   chrome.storage.sync.get([skillsArrayId], logResult);
   getExp(skillName)
   .then( function(exp) {
-    extension_log("Skill " + skillName + " experience value is " + exp);
+    chrome.storage.sync.get([skillName], function(result) {
+      var exp = 0;
```

Comment:
First of all the name of this variable is the same as the function argument. What for is the argument then? If you are not going to use it then replace it's name with '_'. Secondly, try to replace all 'var' keywords with 'let'.

Reply:
You are totally right, this function is a big mishmash. I deleted getExp function and then changed this one function, so it don't use getExp, however I made some mistakes right there.

I deleted getExp function thinking that it is never used besides in "getLevelAndExpNeeded", so I merged those two functions but it looks like getting only exp may be useful. Do you think I should revert this change or keep it as it is?

Why use let instead of vars?

Reply:
'var' keyword is oldfashioned-style JS. In es6 we use let instead of them. The only difference I know is that variables initialized by 'var' are visible in different scope then variables initialized by 'let'. For more information check this: https://stackoverflow.com/questions/762011/whats-the-difference-between-using-let-and-var-to-declare-a-variable